app/views/console/database/collection.phtml

```diff
@@ -568,8 +568,8 @@ $logs = $this->getParam('logs', null);
 
                     <ul class="margin-bottom-large text-fade text-size-small">
                         <li class="margin-bottom-small"><i class="icon-angle-circled-right margin-start-tiny margin-end-tiny"></i> <button data-ls-ui-trigger="open-json" class="link text-size-small">View as JSON</button></li>
-                        <li class="margin-bottom-small"><i class="icon-angle-circled-right margin-start-tiny margin-end-tiny"></i> Last Updated: <span data-ls-bind="{{project-collection.dateUpdated|dateText}}"></span></li>
-                        <li class="margin-bottom-small"><i class="icon-angle-circled-right margin-start-tiny margin-end-tiny"></i> Created: <span data-ls-bind="{{project-collection.dateCreated|dateText}}"></span></li>
+                        <li class="margin-bottom-small"><i class="icon-angle-circled-right margin-start-tiny margin-end-tiny"></i> Last Updated: <span data-ls-bind="{{project-collection.$updatedAt|dateText}}"></span></li>
```

Comment:
Suggestion: Show updatedAt and createdAt just like we show other `$` attributes:

![CleanShot 2022-06-20 at 13 46 27](https://user-images.githubusercontent.com/19310830/174594979-44fdb6cc-7b1c-4385-aa08-87210929c310.png)

Reply:
Let's show the date then. Maybe a different design to better indicate that timestamp would be copied? I think we have to show it, at least in document view if nowhere else. I would maybe even say it's valuable in the list of documents.